ddp-workspace/projects/ddp-rarex/src/styles/_common.scss

```diff
@@ -117,3 +117,9 @@ table {
     vertical-align: top;
   }
 }
+
+ddp-sign-in-out {
+  button {
+    padding: 5px !important;
+  }
```

Comment:
Ticket complained that Login/Logout was difficult to click.
Underneath, the signin-signout component is a button. The inherited padding was 0. I tried to fatten it out, 5px all around.
Note that one consequence is that the dark bar where the login/logout/dashboard links show has also been fattened, but still looks consistent with bar in footer.
On top is the new fatter bar with the easier to click login/logout button, on the bottom the old version:
![Screen Shot 2021-05-10 at 3 56 08 PM](https://user-images.githubusercontent.com/29984380/117717175-7924a980-b1a8-11eb-8da6-6ee88da7545a.png)